lab-miketreat/model/kid-toy.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const debug = require('debug')('http:kid-toy');
+const uuid = require('node-uuid');
+
+module.exports = function(name, type, hazard, id) {
+  if(!name || !type) throw new Error('Invalid arguments');
+
+
+
+  this.name = name;
+  this.type = type;
+  this.hazard = hazard;
+  this.id = id || 1;
+  // this.id = uuid();
```

Comment:
Oops, I changed this line when I was testing everything in the command line so I didn't have to copy and paste a giant ID string and forgot to change it back! Thanks!